lib/ruby_lsp/executor.rb

```diff
@@ -692,6 +727,12 @@ def initialize_request(options)
         )
       end
 
+      signature_help_provider = if enabled_features["signatureHelp"]
+        Interface::SignatureHelpOptions.new(
+          trigger_characters: ["(", " ", ","],
```

Comment:
Should we also trigger on `:` like if I'm typing `bar(a:` wouldn't the help disappear?

Reply:
It's already triggered automatically. It seems editors consider `:` as part of identifiers.

Reply:
Should we add a comment about this for future us?